.github/workflows/release.yml

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Release Che Server
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The version that is going to be released. Should be in format 7.y.z'
+        required: true
+        default: ''
+      releaseParent:
+        description: 'If true, will also release parent of the same version'
+        required: true
+        default: 'false'
+      forceRecreateTags:
+        description: 'If true, tags will be overriden and regenerated. Use with caution'
+        required: false
+        default: 'false'
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+          path: che
+      - uses: actions/checkout@v2
+        if: ${{ github.event.inputs.releaseParent == 'true' }}
+        with:
+          repository: eclipse/che-parent
+          token: {{ secrets.CHE_BOT_GITHUB_TOKEN }}
+          path: che-parent
+      - name: Check existing tag
+        run: |
+          if [[ "$FORCE_RECREATE_TAGS" == "false" ]] && [[ $(cd che && git ls-remote --exit-code origin refs/tags/${{ github.event.inputs.version}} ]]; then
+            echo "cannot create release, when tag already exists"
+            exit 1
+          fi
+      - name: Login to docker.io
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+          registry: docker.io
+      - name: Login to quay.io
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+          registry: quay.io
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.7'
+          java-package: jdk
+          architecture: x64 
+      - name: Set up environment
+        run: |
+          # install more repos
+          sudo apt-get update -y || true
+          # install more dependencies
+          sudo apt-get -y -q install wget curl bash git 
+          java -version
+
+          # want git >=2.24, hub >=2
+          hub --version # hub reports git version too
+
+          # want >=5
+          bash --version
+
+          java -version
+          which java
+          javac -version
+          which javac
+
+          # do we need any of these?
```

Comment:
if these were commented out in the last che 7.25.x releases, we can probably remove them, right?